docusaurus/docs/learn/core-concepts/security/sessions.md

```diff
@@ -1,12 +1,115 @@
+connection requests initiated by Ziti SDKs. API Sessions remain valid as long they have not timed out.
+
+An API Sessions:
+
+- can and are represented by a JSON data structure returned from the Client and Management APIs
+    - returned from:
+        - `POST /edge/management/v1/authenticate`
+        - `GET /edge/management/v1/current-api-session`
+        - `POST /edge/client/v1/authenticate`
+        - `GET /edge/client/v1/current-api-session`
+- can be referenced by an internal `id` and a security token that is in the format of a UUID
+    - the `id` can be used on the following endpoints:
+        - `GET /edge/management/v1/api-sessions/<id>`
+        - `DELETE /edge/management/v1/api-sessions/<id>`
+
+API Sessions are defined in the Client and Management Open API 2.0 specifications under `currentApiSessionDetail`.
+
+Example `POST /edge/management/v1/authenticate` response:
+
+```json
+{
+  "data": {
+    "_links": {
+      "self": {
+        "href": "./api-sessions/cl4zptpgqcrinn0hhgm7ek5ve"
+      },
+      "sessions": {
+        "href": "./api-sessions/cl4zptpgqcrinn0hhgm7ek5ve/sessions"
+      }
+    },
+    "createdAt": "2022-06-29T14:51:07.946Z",
+    "id": "cl4zptpgqcrinn0hhgm7ek5ve",
+    "tags": {},
+    "updatedAt": "2022-06-29T14:51:07.946Z",
+    "authQueries": [
+      {
+        "format": "alphaNumeric",
+        "httpMethod": "POST",
+        "httpUrl": "./authenticate/mfa",
+        "maxLength": 6,
+        "minLength": 4,
+        "provider": "ziti",
+        "typeId": "MFA"
+      }
+    ],
+    "authenticatorId": "vxtlfvUj6",
+    "cachedLastActivityAt": "2022-06-29T14:51:07.945Z",
+    "configTypes": [],
+    "identity": {
+      "_links": {
+        "auth-policies": {
+          "href": "./auth-policies/default"
+        },
+        "authenticators": {
+          "href": "./identities/vxtlfvUj6/authenticators"
+        },
+        "edge-router-policies": {
+          "href": "./identities/vxtlfvUj6/edge-routers"
+        },
+        "enrollments": {
+          "href": "./identities/vxtlfvUj6/enrollments"
+        },
+        "failed-service-requests": {
+          "href": "./identities/vxtlfvUj6/failed-service-requests"
+        },
+        "posture-data": {
+          "href": "./identities/vxtlfvUj6/posture-data"
+        },
+        "self": {
+          "href": "./identities/vxtlfvUj6"
+        },
+        "service-policies": {
+          "href": "./identities/vxtlfvUj6/service-policies"
+        }
+      },
+      "entity": "identities",
+      "id": "vxtlfvUj6",
+      "name": "Default Admin"
+    },
+    "identityId": "vxtlfvUj6",
+    "ipAddress": "127.0.0.1",
+    "isMfaComplete": false,
+    "isMfaRequired": true,
+    "lastActivityAt": "2022-06-29T14:51:07.945Z",
+    "token": "44a20395-1a0e-469d-ad9b-80df8dbbf8c4",
+    "expirationSeconds": 1800,
+    "expiresAt": "2022-06-29T15:21:07.945Z"
+  },
+  "meta": {}
+}
+```
+
+## Session
```

Comment:
This causes me to realize that the way we're naming or explaining "API Session" vs. "Session" needs a little work, mainly "Session." 

"API Session" is the more familiar concept because it's encountered early in the learning curve in several places like running the CLI and using the RESTful Management API. 

"Session" might be referred to as a "Service Session", "Service Chit",  "Service Authorization", or "Service Access." "Session" alone isn't quite meaningful enough, especially because it's a neighboring concept to "API Session."

At a bare minimum, we should uniformly capitalize "Session" to express that it is a Ziti primitive.